nautobot/extras/tests/test_filters.py

```diff
@@ -518,14 +579,10 @@ def test_content_type_id_and_object_id(self):
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
 
 
-class JobFilterSetTestCase(TestCase):
+class JobFilterSetTestCase(FilterTestCases.FilterTestCase):
```

Comment:
`Job` contains a slug although appears to be missing a test. Could this inherit from `FilterTestCases.NameSlugFilterTestCase`?

Reply:
Will give it a try, good catch.